checks.d/mcache.py

```diff
@@ -162,18 +163,18 @@ class Memcache(AgentCheck):
     SERVICE_CHECK = 'memcache.can_connect'
 
     def get_library_versions(self):
-        return {"memcache": memcache.__version__}
+        return {
+            "memcache": pkg_resources.get_distribution("python-binary-memcached").version
+        }
 
     def _get_metrics(self, client, tags, service_check_tags=None):
         try:
-            raw_stats = client.get_stats()
+            stats_by_host = client.stats()
```

Comment:
Took a look at the reference docs (http://python-binary-memcached.readthedocs.io/en/latest/bmemcached/#module-bmemcached.client), `stats()` will return a dict, and `stats(key=None)` suggests the parameter would imply grabbing the stats for a certain key.... 

Please correct me if I'm wrong, but wouldn't we expect this call to return stats for all keys if `key is None` and thus having multiple keys the `len(stats_by_host)` could be greater than 1?